src/interfaces/IFactory.sol

```diff
@@ -52,6 +52,7 @@ interface IFactory {
     event NewFarm(Farm[] farms);
     event UpdateFarm(uint id, Farm farm);
     event SetStrategyAvailableInitParams(string id, address[] initAddresses, uint[] initNums, int24[] initTicks);
+    event AliasNameChanged(address indexed operator, address indexed tokenAddress, string newAliasName);
 
```

Comment:
add your author tag to interface natspec

Reply:
thank you for your review. I have fixed the issues.